docs/site/best-practices.html

```diff
@@ -62,11 +62,11 @@ <h3 class="text-center highlight-heading">Feedback received during pilot phase f
                         <li class="list-group-item"><strong>Integration:</strong> Assignments felt more beneficial to students when presented as part of a larger process. It may be more useful for students to use ComPAIR to see what their peers have done and reflect on their own work in the context of a bigger goal, rather than apply this to standalone assignments that are not concretely tied into the course in any other way.</li>
                     </ul>
                     
-                    <h2>Notes on scoring and grades</h2>
+                    <h2><a id="scoring"></a>Notes on scoring and grades</h2>
```

Comment:
fyi you don't need to use a link tag to anchor. you can use ids attached to any element (so you could just move the id attribute to the h2 tag)